pages/activate/[uid]/[token]/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { appGetLayout } from "components/AppRoute";
+import Signup from "features/auth/signup/Signup";
+import { AUTH, GLOBAL } from "i18n/namespaces";
+import { translationStaticProps } from "i18n/server-side-translations";
+import { GetStaticPaths } from "next";
+
+export const getStaticPaths: GetStaticPaths = () => ({
+  paths: [],
+  fallback: "blocking",
+});
+
+export const getStaticProps = translationStaticProps([GLOBAL, AUTH]);
+
+export default function SignupPage() {
+  return <Signup />;
```

Comment:
I've been thinking about this and it seems wrong to be re-using the `Signup` component here with new API sign up flow, as this seems to me a standalone action more similar to how the `complete-password-reset` page current works, where you'd show the user they have successfully activated their account and that's it.

Reply:
Once Hamish gets around adding the step computation logic, I think it'd make less sense to manipulate the "flow state" in the frontend code ourselves. Instead, we should be reading the result coming back from the API (either it'll be available in the response of the PATCH request, or we might have to do a re-look up again).

Reply:
We don't need to wait for any changes to do this though right? I can refactor activation to its own component and redirect to the homepage instead. I think it would still unblock the work to port login/authentication, even if there's more work to do on the signup flow.